Components/HeroSection.tsx

```diff
@@ -47,7 +47,7 @@ const HeroSection = () => {
           <h1 className="text-black text-8xl w-fit font-archivo">
             Revolutionizing Education in India
           </h1>
-          <div className="flex flex-row items-start gap-4 mt-10">
+          <div className="pad flex flex-row items-start gap-4 mt-10">
```

Comment:
don't use custom classes. As we are using `tailwindcss` please check the documentation and add the desired class here from `tailwindcss` classes